.github/workflows/release-build-integration.yml

```diff
@@ -0,0 +1,107 @@
+name: Build integration release
```

Comment:
Mir fehlt noch bisschen der Überblick, soll die neue Komponente mit dem Rest der Management-Komponente mitdeployed werden? Und wenn ja wurde es schon irgendwie umgesetzt, dass das Backend kommunizieren kann bzw. gibt es schon ein Deployment, was die Images mit dem Tag pullen?

Also grob gesagt, wie kommt die neue EAI Komponente in die Umgebungen und wie wird sie da ausgeführt?

Reply:
@mirrodi Da hast Du mich jetzt erwischt. Da die Komponente ja nur ein Starter ist, gibt es dazu natürlich kein Docker-Image, sondern das Publishing erfolgt über maven-central. Da muss ich den Workflow nochmal  neu aufsetzen.

Reply:
https://github.com/it-at-m/itm-java-codeformat/blob/main/.github/workflows/release.yaml
hier wäre ein Beispiel

Reply:
Müsste man dann noch den Starter irgendwo in einer POM hinzufügen?

Reply:
Ich bin gerade am klären wegen den Nexus Zugangsdaten, aber da gab es mal was

Reply:
[Workflow](https://github.com/it-at-m/digiwf-core/pull/1492/files#diff-159cd13c2f66a13f5708d7000aacc43e6e606310671f484766bd3405036584c4)
[POM](https://github.com/it-at-m/digiwf-core/pull/1492/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8)

Da gibt es einen Beispiel wie es mit dem Central Portal ausschauen sollte. Vielleicht hilft das